doc/.wordlist.txt

```diff
@@ -189,7 +190,10 @@ preseed
 proxied
 proxying
 PTS
+qdisc
+qdisc
```

Comment:
duplicate

Reply:
But you probably need "qdiscs" - aspell isn't smart when it comes to words that are defined here. :(